packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -93,14 +93,14 @@ export default class AutomateProvider extends GenericProvider {
     });
   }
 
-  async percyScreenshotEnd(name, percyScreenshotUrl, statusMessage = null) {
+  async percyScreenshotEnd(name, error) {
     return await TimeIt.run('percyScreenshotEnd', async () => {
       try {
         await this.browserstackExecutor('percyScreenshot', {
           name,
-          percyScreenshotUrl,
-          status: percyScreenshotUrl ? 'success' : 'failure',
-          statusMessage,
+          percyScreenshotUrl: this.buildInfo.url,
```

Comment:
Just thinking if this should be `null` safe.

Reply:
Done